MHTimer.js

```diff
@@ -693,6 +696,20 @@ function parseUserMessage(message) {
             }
             break;
 
+        case 'reset':
+            if (message.author.id === settings.owner) {
+                if (!tokens.length) {
+                    message.channel.send("I don't know what to reset.");
```

Comment:
It should only respond in such a way to me (the owner) since there's no else and then it will continue to the default case for non-owners instead of ignoring their invalid keyword